### PR TITLE
Bacon!

### PR DESCRIPTION
**Pull request by Viz-Titan**
Added raw bacon, bacon related items and recipes, bacon spawning in meat locker, and orderable raw bacon and mixed colored light tubes from supply.

### CODE REVIEW

File: code/modules/food/recipes_microwave_ds.dm [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+	result = /obj/item/weapon/reagent_containers/food/snacks/bacon
+
+/datum/recipe/microwave/baconcheeseburger
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/cheeseburger,
+		/obj/item/weapon/reagent_containers/food/snacks/bacon
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/baconcheeseburger
+
+/datum/recipe/microwave/baconcheeseburger
```
Comment (DTraitor) [inline code/modules/food/recipes_microwave_ds.dm]: 2 recipes shouldn't have the same type

File: code/modules/food/recipes_microwave_ds.dm [hunk 2]
```diff
@@ -1,3 +1,86 @@
...
+/datum/recipe/microwave/deluxebaconcheeseburger
+	fruit = list("cabbage" = 1, "tomato" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/cheeseburger,
+		/obj/item/weapon/reagent_containers/food/snacks/bacon,
+		/obj/item/weapon/reagent_containers/food/snacks/bacon
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/deluxebaconcheeseburger
+
+/datum/recipe/microwave/deluxebaconcheeseburger
```
Comment (DTraitor) [inline code/modules/food/recipes_microwave_ds.dm]: same thing here

**Review by DTraitor — CHANGES_REQUESTED** [review]
Just a few tweaks

File: code/modules/food/recipes_microwave_ds.dm (hunk 1 above)
Comment (Viz-Titan) [reply code/modules/food/recipes_microwave_ds.dm]: Oh ok I was going for a multiple recipe approach but forgot to even test both versions in game. When it likely fails, could of maybe looked at the combinable burger or sandwich code or something. I'll get on it, or will just choose which recipe to go with for the moment.

**Review by DTraitor — APPROVED** [review]
Looks good to me

**Pull request merged**